### DISCUSSION TITLE
Navidrome docker migrate data from one host to another

### DISCUSSION THREAD

**Discussion by binamov** [navidrome/navidrome#2732] [Q&A] [answered]
Hi folks,

I've been running Navidrome in a docker container, with a docker volume `navidrome` bound to `/data`.

The one big difference is on the old system user:gid was 1001:1001, on the new system it's 0:0 - root owns everything.

I'm moving to a new physical server, rsync'ed the navidrome.db from the old host to the new host as root, so everything is owned by root. Locations as bindings are still /music as far as navidrome is concerned. When starting docker i see in logs:

```
 _   _             _     _
| \ | |           (_)   | |
|  \| | __ ___   ___  __| |_ __ ___  _ __ ___   ___
| . ` |/ _` \ \ / / |/ _` | '__/ _ \| '_ ` _ \ / _ \
| |\  | (_| |\ V /| | (_| | | | (_) | | | | | |  __/
\_| \_/\__,_| \_/ |_|\__,_|_|  \___/|_| |_| |_|\___|
                          Version: 0.50.2 (823bef54)
time="2023-12-21T19:05:00Z" level=error msg="Error disabling foreign_keys" error="malformed database schema (/music/2019｜Space Dimension Controller - Redemption Of The Cryonauts｜Not On Label REDEMPTION｜Vinyl 16 - 48/1.04 - Phase Pod Dispersal.flac)"

time="2023-12-21T19:05:00Z" level=fatal msg="Database could not be opened!" error="malformed database schema (/music/2019｜Space Dimension Controller - Redemption Of The Cryonauts｜Not On Label REDEMPTION｜Vinyl 16 - 48/1.04 - Phase Pod Dispersal.flac)"
```
Any suggestions on how I can migrate users and cache and playlists and all my data?

**Comment by binamov** [comment]
Navidrome image on new and old systems is not identical, but they're not very far off, i doubt that's the issue

```
new$ docker image list
deluan/navidrome            latest    6501b86eeb45   7 days ago      170MB

old$ docker image list
deluan/navidrome                         latest         df130a638d0f   4 weeks ago     174MB
```

**Comment by binamov** [comment, marked as answer]
i got it work by:
1. rsync music over 
2. letting new navidrome complete build up of its cache by watching the docker logs
3. stop both instances, rsync navidrome.db from old to new
4. start the new instance

**Comment by g4njawizard** [comment]
Thank you for sharing!
Just ran into the same issue.
